packages/opentelemetry-exporter-collector/src/transform.ts

```diff
@@ -133,14 +133,20 @@ export function toCollectorEvents(
 /**
  * Converts links
  * @param span
+ * @param useHex - if ids should be kept as hex without converting to base64
  */
-export function toCollectorLinks(
-  span: ReadableSpan
+function toCollectorLinks(
+  span: ReadableSpan,
+  useHex?: boolean
```

Comment:
Approving, but would be nice if you can add a test for this.

Reply:
done